jenkins/workers.py

```diff
@@ -0,0 +1,94 @@
+"""
+Use the jenkins queue and computer APIs to log info about workers.
+
+Usage:
+    `python jenkins/workers.py -j https://example.jenkins.com`
+
+Example Output:
+    [INFO] Counts of workers and queue length from Jenkins API:
```

Comment:
I could totally be wrong here but I thought that it's the jenkins display that adds the timestamps. The [raw view](https://build.testeng.edx.org/job/edx-platform-test-subset/77858/consoleText) doesn't have them. 
Though I also don't know how the display could add them in. Probably by magic.